dotnet/src/webdriver/WebDriver.nuspec

```diff
@@ -36,12 +36,6 @@
     </frameworkAssemblies>
   </metadata>
   <files>
-    <file src="**\WebDriver.dll" target="lib" />
-    <file src="**\WebDriver.pdb" target="lib" />
-    <file src="**\WebDriver.xml" target="lib" />
-    <file src="..\icon.png" target="images" />
-    <file src="..\manager\**" target="manager" />
-    <file src="..\Selenium.WebDriver.targets" target="build" />
-    <file src="..\Selenium.WebDriver.targets" target="buildTransitive" />
+    <file src="**" target="." exclude="obj\**;*.csproj;*.config"/>
```

Comment:
Better to declare package's content as it was before, in more precise way. Now it's something unmanageable, unpredictable content.

Reply:
The `nupsec` files are written to work with the bazel build, and aren't designed to be used with the regular .Net tooling. The content is completely predictable, since we construct the contents hermetically.

Reply:
```
<file src="**" target="." exclude="obj\**;*.csproj;*.config"/>
```

I don't think it's predictable content. Everything what is on disk now will be packed, it is not something we expect. Just explore nupkg file, you might find extra files there.